kahuna/public/js/preview/image.js

```diff
@@ -82,6 +82,25 @@ image.controller('uiPreviewImageCtrl', [
           `${window._clientConfig.staffPhotographerOrganisation}-owned: ${ctrl.image.data.metadata.description}` :
           ctrl.image.data.metadata.description;
 
+      ctrl.image.isPotentiallyGraphic = $rootScope.shouldBlurGraphicImages && (ctrl.image.data.isPotentiallyGraphic || !![
+        'graphic content',
+        'depicts death',
+        'dead child',
+        'child casualty',
+        'sensitive material',
+        'injured',
+        'wounded',
+        'killed',
+        'body of',
+        'bodies of',
+        'smout'
```

Comment:
Is "smout" another word for child? Not sure that needs to be in the list? (maybe you were using that as a test though?